### PR TITLE
Lab1

### PR DESCRIPTION
**Pull request by JustSuHaR1K**

### CODE REVIEW

File: Lab1/Services/CSVReader.cs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                            catch (TypeConverterException)
+                            {
+                                logger.Error($"Student's {csv.Context.HeaderRecord[0]} mark of {csv.Context.HeaderRecord[index]} can't be converted in integer");
+                                throw;
+                            }
+                            catch (Exception)
+                            {
+                                logger.Error("Unexpected error");
+                                throw;
+                            }
```
Comment (alexey62soft) [inline Lab1/Services/CSVReader.cs:68]: You can use a single exception instead of a couple.

File: Lab1/Services/JsonWriter.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
+﻿using Lab1.Interfaces;
+using Lab1.Models;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Lab1.Services
+{
+    class JsonWriter : IWriter
```
Comment (alexey62soft) [inline Lab1/Services/JsonWriter.cs:10]: Where is access modificator?

File: Lab1/Services/ExcelWriter.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
+﻿using Lab1.Interfaces;
+using Lab1.Models;
+using OfficeOpenXml;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Lab1.Services
+{
+    class ExcelWriter : IWriter
```
Comment (alexey62soft) [inline Lab1/Services/ExcelWriter.cs:9]: Modificator?

**Comment by alexey62soft** [comment]
Still not all issues resolved.

**Comment by alexey62soft** [comment]
Any updates?

**Comment by alexey62soft** [comment]
@JustSuHaR1K please, add lab report to the PR.

**Comment by JustSuHaR1K** [comment]
But I have Lab1.docx file, this is the report.

**Comment by JustSuHaR1K** [comment]
[Lab1.docx](https://github.com/JustSuHaR1K/SVIATASLAU-KAVALIOU-DotNet/files/4580408/Lab1.docx)

**Comment by alexey62soft** [comment]
Oh, I see it.

**Pull request merged**